spring-integration-core/src/main/java/org/springframework/integration/transformer/SyslogToMapTransformer.java

```diff
@@ -54,7 +55,8 @@
 
 	public static final String UNDECODED = "UNDECODED";
 
-	private final Pattern pattern = Pattern.compile("<([^>]+)>(.{15}) ([^ ]+) (?:([^:]+): )?(.*)", Pattern.DOTALL);
+	private final Pattern pattern = Pattern.compile("<([^>]+)>(.{15}) ([^ ]+) ([a-zA-Z0-9]{0,32})(.*)",
```

Comment:
I doubt it will make any difference at runtime and, personally, I think the existing code is clearer.